packages/vaadin-combo-box/test/helpers.js

```diff
@@ -1,4 +1,6 @@
-export const IOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
+import { click, mousedown, mouseup } from '@vaadin/testing-helpers';
+
+export const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
Consider using [`isDesktopSafari`](https://github.com/vaadin/testing-helpers/blob/a42f51c3218326ba5bb10ec1ed2e1450a8beca26/src/browser.ts#L5-L6) helper from `@vaadin/testing-hepers` here, it should cover WebKit.

Reply:
Thanks! Fixed.